src/apps/angle/ethereum/angle.agtoken.token-fetcher.ts

```diff
@@ -0,0 +1,87 @@
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
+import { getImagesFromToken } from '~app-toolkit/helpers/presentation/image.present';
+import { ContractType } from '~position/contract.interface';
+import { BalanceDisplayMode } from '~position/display.interface';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { ANGLE_DEFINITION } from '../angle.definition';
+import { AngleContractFactory } from '../contracts';
+import { fetchTokenList } from '../helpers/angle.api';
+
+const appId = ANGLE_DEFINITION.id;
+const groupId = ANGLE_DEFINITION.groups.agtoken.id;
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.TokenPositionFetcher({ appId, groupId, network })
+export class EthereumAngleAgtokenTokenFetcher implements PositionFetcher<AppTokenPosition> {
```

Comment:
I don't think we need this token fetcher; Zapper already shows `agEUR` as a base token: https://zapper.fi/account/0x735a26a57a0a0069dfabd41595a970faf5e1ee8b